### DISCUSSION TITLE
iii - proposal, candidate mining

### DISCUSSION THREAD

**Discussion by teavar** [idena-network/idena-docs#158] [General]
iii - proposal, candidate mining

the idea is to allow candidates to mine on their stand alone nodes as soon as they activate their invitations. All funds go into the stake wallet and get released as usual, i.e. when becoming verified.

**Comment by NPC69** [comment]
Interesting

**Comment by teavar** [comment]
also a requirement could be a hard level certificate above some high %
https://discord.com/channels/634481767352369162/634481767843364866/1129069622239428669

**Comment by ltraveler** [comment]
Good idea to be sensible of the fact that a candidate usually has 0 iDNA at stake, making it akin to free mining. I am certain that there will be individuals who are expected to replenish their stake during the candidate status, only to end up losing everything due to validation failure.

**Reply by teavar** [reply]
If by free mining you mean unpaid then that's not correct, they will get paid for the mining into the stake wallet as I stated in the original message.
Now as far as candidates pumping up their stakes and then losing the stake in case of a failed validation, well, that's their problem, they should read how the idena works before investing their own funds (out of greed or the impulse of making a quick buck)

**Reply by ltraveler** [reply]
> If by free mining you mean unpaid then that's not correct

I mean, people can already start mining once they've obtained the newbie status. However, during the candidate status, they don't have any coins at stake. The only way for them to earn money in case of your proposal is by adding coins to their stake, which they can do by buying them in the exchange market, for example. However, it can be quite dangerous because, as far as I know, the inviter could destroy the invitee identity. I'm not even sure if the current version of the protocol supports stake replenishment for the candidate status.

**Reply by teavar** [reply]
> The only way for them to earn money in case of your proposal is by adding coins to their stake, 

Candidates would mine starting with 0 stake but 100% of the mined coins would go into the stake wallet. Candidates would not be able to replenish the stake from the main wallet (e.g. if they buy idnas on exchanges). Of course the protocol would have to be modified.

> However, it can be quite dangerous because, as far as I know, the inviter could destroy the invitee identity.


Yep, valid point, the invitor can revoke the invitation. Protection would also need to be implemented into the protocol. I've got some ideas but the modification to the protocol would probably be pretty complex. It would be easier to add new stuff if idena had extra types of wallets.
